package-lock.json

```diff
@@ -41762,6 +41763,17 @@
         "balanced-match": "^1.0.0"
       }
     },
+    "node_modules/typedoc/node_modules/marked": {
```

Comment:
The 'marked' package version under 'typedoc' dependencies is still at 4.3.0. If the package version is upgraded globally, consider upgrading it here as well to maintain consistency.